src/plugins/system-greeting/index.js

```diff
@@ -0,0 +1,83 @@
+const Periodic = require( "Periodic" );
+const Listener = require( "Listener" );
+
+class SystemGreeting
+{
+    constructor( name, deps )
+    {
+        console.log( "SystemGreeting plugin loaded" );
+
+        this.globalBus        = deps.globalEventLoop;
+        this.cockpitBus       = deps.cockpit;
+
+        var wakeAttempts      = 0;
+        var maxWakeAttempts   = 10;
```

Comment:
I don't believe it should be user configurable. I could move the max out of the class as a constant for clarity.